web/src/engine/osk/src/input/gestures/specsForLayout.ts

```diff
@@ -443,7 +443,7 @@ export function longpressContactModel(params: GestureParams, enabledFlicks: bool
       duration: spec.waitLength,
       expectedResult: true
     },
-    validateItem: (key: KeyElement) => !!key?.key.spec.sk,
+    validateItem: (_: KeyElement, baseKey: KeyElement) => !!baseKey?.key.spec.sk,
```

Comment:
https://github.com/keymanapp/keyman/blob/4755be9b869a8bf23a2da9d4671e711a5e35d432/common/web/gesture-recognizer/src/engine/headless/gestures/specs/contactModel.ts#L26-L36

The issue was obvious once I saw it.